src/sidebar/util/typing.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Helper functions for cases in which logically a reference is definitely
```

Comment:
```suggestion
 * Helper function for cases in which logically a reference is definitely
```

Since this JSDoc refers to one specific function rather than the whole module.